build.gradle

```diff
@@ -41,6 +41,12 @@ subprojects {
         annotationProcessor "com.querydsl:querydsl-apt:5.0.0:jakarta"
         annotationProcessor "jakarta.annotation:jakarta.annotation-api"
         annotationProcessor "jakarta.persistence:jakarta.persistence-api"
+
+        //WebFlux 주입
+        implementation 'org.springframework.boot:spring-boot-starter-webflux'
+
+        //netty-resolver-dns-native-macos 의존성 주입
+        implementation 'io.netty:netty-resolver-dns-native-macos:4.1.113.Final:osx-aarch_64'
```

Comment:
@PNoahKR 
이건 어느부분에서 발생하는 어떤오류때문에 추가된건가요?

Reply:
@evan-song2 API통신을 할때 Consoled에서
`2024-09-24T17:39:00.768+09:00 ERROR 28339 --- [nio-8080-exec-2] i.n.r.d.DnsServerAddressStreamProviders  : Unable to load io.netty.resolver.dns.macos.MacOSDnsServerAddressStreamProvider, fallback to system defaults. This may result in incorrect DNS resolutions on MacOS. Check whether you have a dependency on 'io.netty:netty-resolver-dns-native-macos'. Use DEBUG level to see the full stack: java.lang.UnsatisfiedLinkError: failed to load the required native library`
라는 에러가 떠서 찾아봤는데, 
https://shanepark.tistory.com/495
위 링크에서 오류에 대한 방안으로 Netty DNS Resolver 의존성을 주입해 봤을때 에러메세지 없이 뜨는 것으로 나와 주입했습니다.
<img width="1822" alt="image" src="https://github.com/user-attachments/assets/a04308ca-3a0a-4306-80fe-98ab205a37ec">